ExtDataValuesOnly.md

```diff
@@ -0,0 +1,163 @@
+---
+RFC: 26
+Status: Proposed
+---
+
+# Title
+External data storage (values only)
+
+## Abstract
+Currently Valkey allows to store data in memory only. The option to store data externally (on local or network disk, e.g. SSD) is proposed here.
+Only values are about to be stored externally - option to store both keys and values externally is proposed in another RFC (#25).
+
+## Motivation
+1) Cost saving - disks are cheaper than memory.
+2) Larger storage - memory ends up fast in some use cases.
+3) NOT a data durability enhancement - this is out of scope of this initiative.
+
+## Design considerations
+0) The whole thing on this stage is about strings only - it's not about the impossibility or lack of demand for other Valkey data structures (sets, hashes, etc.), it's about the scope. Other data structures are supposed to be added to scope based on community requests after successful implementation of the feature for strings.
+1) Keys are just stored as usual; values are stored neither in Valkey dict, nor in any Valkey directly controlled memory space.
+2) Values externally stored are interacted via module - exact implementation of how this is done is up to module creator.
+
+## Specification
+VC<N> = Valkey config option with its number for reference between RFC parts
+VF<N> = flag for existing Valkey command with its number for reference between RFC parts
+* Passing value storage<->client should be made in a way to prevent OOM during loading large data to limited memory.
+
+### GET flow
+1) Client requests a key.
+2) Valkey dict contains the key with attribute "memory-stored" OR missing dict when requested without flag (VF1) - end of story.
+3*) Key attribute is "external-stored" - value is requested from external storage.
```

Comment:
```suggestion
3*) Key attribute is "external-stored" - value is requested from external storage.
```
How is it requested and delivered from external storage? Will we hand off the client to the module? We can't block the main thread.